tunder/lib/src/console/commands/migrations/make_migration_command.dart

```diff
@@ -1,27 +1,34 @@
 import 'dart:io';
 
 import 'package:clock/clock.dart';
+import 'package:intl/intl.dart';
 import 'package:path/path.dart' as path;
 import 'package:tunder/console.dart';
+import 'package:tunder/src/console/commands/migrations/contracts/migration_command.dart';
 import 'package:tunder/tunder.dart';
 
-class MakeMigrationCommand extends Command {
+class MakeMigrationCommand extends MigrationCommand {
   final name = 'make:migration';
   final description = 'Create a migration file';
   final String stubsDir;
   final String destinationDir;
 
   String? _migrationName;
   String get migrationName => _migrationName ??= argResults!.rest.join(' ');
-  int? _timestamp;
-  int get timestamp => _timestamp ??= clock.now().millisecondsSinceEpoch;
+
+  String? _id;
+  String get id {
+    if (_id != null) return _id!;
+
+    return _id = DateFormat('yyyy_MM_dd_HHmmss').format(clock.now());
+  }
```

Comment:
The `late` keyword can accomplish the same behavior.
```dart
late String id = DateFormat('yyyy_MM_dd_HHmmss').format(clock.now());
```

Reply:
Yeah, but I should initialize the id in the `run` call or in the constructor. But it's a good suggestion though. I'll keep it in mind in future implementations since I still have some code styling from PHP and Javascript. For now, if I change this I will need to change all the tests as well because it's a time-sensitive testing.